src/Swagger/DocumentationNormalizer.php

```diff
@@ -208,6 +176,33 @@ public function normalize($object, $format = null, array $context = [])
         return $doc;
     }
 
+    private function getPath(string $resourceClass, ResourceMetadata $resourceMetadata, string $operationName, bool $collection)
+    {
+        // Check if the operation has a custom path
+        $method = $collection ? 'getCollectionOperationAttribute' : 'getItemOperationAttribute';
+        $path = $resourceMetadata->{$method}($operationName, 'path');
+        if (null !== $path) {
+            return $path;
+        }
+
+        if (null === $this->iriConverter) {
+            throw new InvalidArgumentException(sprintf('%s can\'t manage the %s %s operation of the resource "%s" because it needs an instance of %s to manage standard operations (path not defined manually).', __CLASS__, $collection ? 'collection' : 'item', $resourceClass, IriConverterInterface::class));
+        }
+
+        // @todo Fix the limitation of the IriConverter when a resource
+        // has no collection operations
```

Comment:
@dunglas makes sense :)
I'm not sure yet how we should do that... Maybe creating a new class `OperationPathResolver` would do the trick.